spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cache/CacheProperties.java

```diff
@@ -233,4 +242,72 @@ public void setConfig(Resource config) {
 
 	}
 
+	/**
+	 * Redis-specific cache properties. Properties set will be used as the defaults for
+	 * all Redis caches.
+	 *
+	 * @see org.springframework.data.redis.cache.RedisCacheConfiguration
+	 */
+	public static class Redis {
+
+		/**
+		 * The TTL for entries. By default, entries do not expire, a value of zero
+		 * indicates an eternal entry, and the given value will be converted to seconds..
+		 *
+		 * @see Duration#parse(CharSequence)
```

Comment:
We don't make such `@see` link here. We use the javadoc to infer the description. Can you please remove them all?